packages/snaps-execution-environments/src/webworker/pool/WebWorkerPool.test.browser.ts

```diff
@@ -56,7 +56,7 @@ async function getResponse(
   });
 }
 
-describe('WebWorkerPool', () => {
+describe.skip('WebWorkerPool', () => {
```

Comment:
Disabled this because it's very flaky, and we don't use the `WebWorkerPool` anywhere.